packages/ember-glimmer/tests/integration/input-test.js

```diff
@@ -0,0 +1,155 @@
+import { RenderingTest, moduleFor } from '../utils/test-case';
+import { set } from 'ember-metal/property_set';
+
+moduleFor('Input element tests', class extends RenderingTest {
+  runAttributeTest(attributeName, values) {
+    let template = `<input ${attributeName}={{value}}>`;
```

Comment:
Did you intentionally not quote or could it also have been `="{{value}}"` instead?